client/package.json

```diff
@@ -17,7 +17,9 @@
     "tslint": "^5.0.0",
     "typescript": "^2.2.2"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@types/requirejs": "^2.1.29"
```

Comment:
You probably want this as a development dependency, since this is only used by `tsc`, and we commit the `js` output.